frontend/online-shop/src/App.tsx

```diff
@@ -0,0 +1,29 @@
+import { ThemeProvider } from "@mui/material";
+import { Container } from "@mui/system";
+import { Route, Routes } from "react-router-dom";
+import "./App.css";
+import Header from "./components/header/header.component";
+import ProductDetail from "./components/product-details/products-details.component";
+import { myTheme } from "./helpers/custom-theme";
+
```

Comment:
Try to always have your imports tied together, no need for empty lines between them